modules/serverless-invoking-quarkus-functions.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-quarkus-functions.adoc
+
+[id="serverless-invoking-quarkus-functions_{context}"]
+= About invoking Quarkus functions
+
+You can create a Quarkus project that responds to CloudEvents, or one that responds to simple HTTP requests. CloudEvents in Knative are transported over HTTP as a POST request, so either function type can listen and respond to incoming HTTP events.
+
+When an incoming request is received, Quarkus functions are invoked with an instance of a permitted type.
+
+.Function invocation options
+[options="header",cols="d,d,m"]
+|====
+|Invocation method |Data type contained in the instance |Example of data
+|HTTP POST request | JSON object in the body of the request |`{ "customerId": "0123456", "productId": "6543210" }`
+|HTTP GET request | Data in the query string |`?customerId=0123456&productId=6543210`
+|`CloudEvent` | JSON object in the `data` property |`{ "customerId": "0123456", "productId": "6543210" }`
+|====
+
+The following example shows a function that receives and processes purchase data, such as the `customerId` and `productId` data in the table:
```

Comment:
I guess, when the above table is rendered as a table, this sentence will be easier to understand, but reading it as source code, this would have helped me.
```suggestion
The following example shows a function that receives and processes purchase data, such as the `customerId` and `productId` data in the table above:
```